src/app/child-dev-project/children/child-details/form/form.component.html

```diff
@@ -0,0 +1,153 @@
+<form [formGroup]="form">
+    <div
+            fxLayout="row wrap"
+            fxLayout.xs="row wrap"
+            fxLayout.md="row wrap"
+            fxLayout.sm="row wrap"
+    >
+        <div *ngFor="let col of config.cols" fxFlex>
+            <div *ngFor="let row of col">
+                <div *ngIf="row.input === 'photo'" class="child-pic-container">
```

Comment:
all the details inside this should actually be its own component to separate this.

We can refactor this later, however